cli/bpmetadata/cmd.go

```diff
@@ -203,3 +239,47 @@ func createContent(bpPath string, rootPath string, readmeContent []byte) Bluepri
 
 	return content
 }
+
+func writeMetadata(obj *BlueprintMetadata) error {
+	// marshal and write the file
+	yFile, err := yaml.Marshal(obj)
+	if err != nil {
+		return err
+	}
+
+	return os.WriteFile(metadataFileName, yFile, 0644)
+}
+
+func unmarshalMetadata(bpPath string) (*BlueprintMetadata, error) {
```

Comment:
nit: lets make this and writeMetadata exported functions so other consumers can use them.

Reply:
ack